tools/tensorflow-quantization/README.md

```diff
@@ -60,17 +60,17 @@ If all tests pass, installation is successful.
 
 ## Documentation
 
-TensorFlow 2.x Quantization toolkit [userguide](http://tensorrt.gitlab-master-pages.nvidia.com/Tools/tensorflow-quantization).
+TensorFlow 2.x Quantization toolkit [userguide](https://docs.nvidia.com/deeplearning/tensorrt/tensorflow-quantization-toolkit/docs/index.html).
 
 ## Known limitations
 
 1. Only Quantization Aware Training (QAT) is supported as a quantization method.
 2. Only Functional and Sequential Keras models are supported. Original Keras layers are wrapped into quantized layers using TensorFlow's [clone_model](https://www.tensorflow.org/api_docs/python/tf/keras/models/clone_model) method, which doesn't support subclassed models.
 3. Saving the quantized version of a few layers may not be supported in `TensorFlow < 2.8`:
    - `DepthwiseConv2D` support was added in TF 2.8.
-   - `Conv2DTranspose` is not yet supported by TF (see the open bug [here](https://github.com/tensorflow/model-optimization/issues/964)).
+   - `Conv2DTranspose` is not yet supported by TF (see the open bug [here](https://github.com/tensorflow/model-optimization/issues/964)). 
        However, there's a workaround if you do not need the TF2 SavedModel file and just the ONNX file:
-       1. Implement `Conv2DTransposeQuantizeWrapper`. See our [User Guide](https://docs.nvidia.com/deeplearning/tensorrt/tensorflow-quantization-toolkit/docs/docs/add_new_layer_support.html#example) for more information on how to do that.
+       1. Implement `Conv2DTransposeQuantizeWrapper`. See our [userguide](https://docs.nvidia.com/deeplearning/tensorrt/tensorflow-quantization-toolkit/docs/docs/add_new_layer_support.html#example) for more information on how to do that.
```

Comment:
```suggestion
       1. Implement `Conv2DTransposeQuantizeWrapper`. See our [user guide](https://docs.nvidia.com/deeplearning/tensorrt/tensorflow-quantization-toolkit/docs/docs/add_new_layer_support.html#example) for more information on how to do that.
```

Reply:
I'm okay with that, we went with `userguide` over `User Guide` (or a variation of that) to match the [pytorch-quantization README](https://github.com/NVIDIA/TensorRT/tree/main/tools/pytorch-quantization#resources). Please let us know which one is preferred.

Reply:
Prefer `user guide`

Reply:
Noted and updated.